src/client/package.json

```diff
@@ -96,14 +96,12 @@
     "@storybook/addon-essentials": "^6.5.13",
     "@storybook/addon-links": "^6.5.13",
     "@storybook/react": "^6.5.13",
-    "@testing-library/jest-dom": "^5.11.10",
     "@testing-library/react": "^11.2.6",
     "@testing-library/react-hooks": "^5.1.1",
     "@testing-library/user-event": "^13.1.1",
     "@types/d3": "^6.3.0",
     "@types/dialogflow": "^0.11.0",
     "@types/google.analytics": "^0.0.42",
-    "@types/jest": "^27.0.0",
     "@types/lodash": "^4.14.178",
     "@types/node": "^14.14.37",
     "@types/openfin": "^51.0.1",
```

Comment:
consider adding `"@vitest/ui"` 
with a script to run the tests in the browser
```
    "test:ui": "vitest --ui",
```
@Clark-Adaptive set this up in RA and it looked pretty nice (the default vitest output is not the most readable, but perfectly functional ofc)

Reply:
leaving this for now ...